src/components/views/voip/VideoFeed.js

```diff
@@ -21,9 +21,23 @@ var React = require('react');
 module.exports = React.createClass({
     displayName: 'VideoFeed',
 
+    componentDidMount() {
+        this.refs.vid.addEventListener('resize', this.onResize);
+    },
+
+    componentWillUnmount() {
+        this.refs.vid.removeEventListener('resize', this.onResize);
+    },
+
+    onResize: function(e) {
+        if(this.props.onResize) {
+            this.props.onResize(e);
```

Comment:
> Can we add a propType for this please?

done c95cd73

> this.refs.vid.addEventListener('resize', this.props.onResize);

actually it strikes me that making the addEventListener conditional would be better yet.